app/Http/Controllers/Multiplayer/RoomsController.php

```diff
@@ -138,12 +138,14 @@ public function leaderboard($roomId)
 
     public function part($roomId, $userId)
     {
+        $currentUser = \Auth::user();
         // this allows admins/host/whoever to remove users from games in the future
-        if (get_int($userId) !== auth()->user()->user_id) {
+        if (get_int($userId) !== $currentUser->getKey()) {
             abort(403);
```

Comment:
Adjusted to follow the same structure as `join()`, which I assume was adjusted for a reason in https://github.com/ppy/osu-web/pull/11786.